components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Query } from '@apollo/client/react/components';
+import { Twitter } from '@styled-icons/fa-brands/Twitter';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import Container from './Container';
+import { Flex } from './Grid';
+import HTMLContent from './HTMLContent';
+import Image from './Image';
+import Link from './Link';
+import Loading from './Loading';
+import StyledButton from './StyledButton';
+import StyledCarousel from './StyledCarousel';
+import StyledLink from './StyledLink';
+import Modal, { ModalBody, ModalFooter, ModalHeader } from './StyledModal';
+import { P, Span } from './Text';
+
+const newsAndUpdatesQuery = gqlV2/* GraphQL */ `
+  query ChangelogUpdates($collectiveSlug: String, $onlyChangelogUpdates: Boolean) {
+    account(slug: $collectiveSlug) {
+      updates(onlyChangelogUpdates: $onlyChangelogUpdates) {
+        nodes {
+          id
+          slug
+          publishedAt
+          title
+          summary
+        }
+      }
+    }
+  }
+`;
+
+const NewsAndUpdatesModal = ({ onClose, ...modalProps }) => {
+  return (
+    <Modal onClose={onClose} width="576px" {...modalProps}>
+      <ModalHeader>
+        <Flex width="100%">
+          <Flex>
+            <Span pt={1}>
+              <Image
+                width={41}
+                height={28}
+                src="/static/images/updates-and-news-modal-icon.svg"
+                alt="Updates and News Icon"
+              />
+            </Span>
+            <P fontSize={['25px', '28px']} fontWeight="500" lineHeight="36px" color="black.900" margin="0px 16px">
+              <FormattedMessage id="NewsAndUpdates.link.newsAndUpdates" defaultMessage="News and Updates" />
+            </P>
+          </Flex>
+        </Flex>
+      </ModalHeader>
+      <hr />
+      <ModalBody>
+        <Query
+          query={newsAndUpdatesQuery}
+          variables={{ collectiveSlug: 'opencollective', onlyChangelogUpdates: true }}
+          context={API_V2_CONTEXT}
+        >
+          {({ data, loading }) =>
+            loading === false && data ? (
```

Comment:
Not for now, but we should have an issue to followup on this: we're going to show an infinite spinner if there's an error with the query. We should rather display this error.

Reply:
@Betree : Ah yes. Will open a new issue; thanks for pointing it out. 😄